Bottom: 339971121c9e29c477cbb2f30cf6dbebc3be20ff
Top:    798d2ba16acf0670b2db9ddb31978ce4b0bd5b3e
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-02-28 15:11:13 +0800

Refresh of virtio-net-re-enable

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 0f12cf2..ca1441d 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -2608,8 +2608,8 @@ static ssize_t mergeable_rx_buffer_size_show(struct netdev_rx_queue *queue,
 	BUG_ON(queue_index >= vi->max_queue_pairs);
 	avg = &vi->rq[queue_index].mrg_avg_pkt_len;
 	return sprintf(buf, "%u\n",
-		get_mergeable_buf_len(&vi->rq[queue_index], avg,
-				      headroom + tailroom));
+		       get_mergeable_buf_len(&vi->rq[queue_index], avg,
+				             headroom + tailroom));
 }
 
 static struct rx_queue_attribute mergeable_rx_buffer_size_attribute =
